pkg/restapi/issuer/operation/operations.go

```diff
@@ -40,6 +41,8 @@ const (
 	// http params
 	idPathParam     = "id"
 	txnIDQueryParam = "txnID"
+	stateQueryParam = "state"
+	redirectURLFmt  = "%s?state=%s&code=%s"
```

Comment:
this looks too similar to an oauth2 request

Reply:
yes (intentional - used similar names).

Reply:
please use different names in order to avoid confusion

Reply:
Sure, please suggest.

Reply:
why isn't an OTP used? the connectionID would not need to leave the adapter

Reply:
Yeah, I was planning to update it as part of next PR when this OTP is really used. For now, I wanted to define the API.

Reply:
`code` should be removed and instead just use the `state` value to correlate the redirect with the callback.

`connectionID` is too stable and is basically an identifier that should never leave the adapter

Reply:
code is for correlating in adapter, I dont want to use state.

Reply:
connectionID should not be leaked

Reply:
https://github.com/trustbloc/edge-adapter/pull/105/files#r443821604